### PR TITLE
documentation: test_flows.py test description

### PR DESCRIPTION
**Pull request by abelog**
There is no description of test which are done in test_flows.py.
So no one have confirmed info about test_flows.py -> Cert test mapping
and general overview of tests are missing.

General intention of this PR is to form up such document in form of `README.md`
in `tests`section. As i am collecting info on it for F2C presentation.

### CODE REVIEW

File: tests/README.md
Comment (arnout) [inline tests/README.md]: What is this?

File: tests/README.md
Comment (arnout) [inline tests/README.md]: I didn't review all of the above -- too boring. I am a bit unsure what value it adds, it seems pretty easy to get the same information by just looking at the source code...

File: tests/README.md
Comment (arnout) [inline tests/README.md]: Maybe we should explicitly indicate the step(s) that are covered by a particular flow.

Maybe we should abbreviate the certification test description a little (can be done with a simple search-replace: `M[AC]UT §[^ ]* Multi-AP ` to nothing; `test |` -> `|`).

```suggestion
| 4.2.1 Ethernet Onboarding and Initialization | step 1-9: -<br>step 10: channel_selection<br>step 11-12: (topology) |
```

Note that (after #965 is merged) I think the channel selection test covers step 10 completely, so no parenthesis any more.

That said, I think we can merge it as is now and fill in details later. Just one or two examples that are worked out in detail to show the direction we want to go to.

**Review by arnout — CHANGES_REQUESTED** [review]
I guess the idea is to squash everything in a single commit before applying? If not, I think you should. The individual commits have little merit and just pollute the history. If you plan on squashing commits, it's a good idea to make sure at least one commit has no Signed-off-by tag - that way, you're sure the merging is blocked until you actually squashed it, because the DCO check will fail. Also, to allow reviewers to validate the commit message as well as the change itself, it's convenient to:
- use fixup commits whenever you don't change the commit message;
- use squash commits with the full new commit message when you do change the commit message;
- (optionally) point out you did this in the PR comments.

Like for #965, I'm going to mark this as changes requested and re-review when the commit messages are updated.

File: tests/README.md
Comment (MuteSpirit) [reply tests/README.md]: Removed

File: tests/README.md
Comment (MuteSpirit) [reply tests/README.md]: Agree
But i suggest to not delete it right now and not broke document structure

File: tests/README.md
Comment (MuteSpirit) [reply tests/README.md]: Apply your code suggestion
And add similar test steps cover description for 4.3.2

**Review by arnout — APPROVED** [review]
You'll also need to change the commit message after squashing the two commits. Everything after the first paragraph can be removed IMO. Oh, and keep two signoffs, both Anton's and Ivan's.

**Review by VladyslavTupikin — APPROVED** [review]

**Pull request merged**
